components/Icons/AvatarIcon.tsx

```diff
@@ -0,0 +1,12 @@
+import { createIcon } from '@chakra-ui/icon'
+
+export const AvatarIcon = createIcon({
```

Comment:
I added `react-feather` in another PR and that ~should~ have all the icons we need so we don't need to add them like this anymore - see their [user icon](https://feathericons.com/?query=user).